### PR TITLE
fix: fix polygon bbox validation bug

### PR DESCRIPTION
**Pull request by asafMasa**
Fix polygon validation when receiving BBOX in Polygon

| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: tests/unit/validator/validationManager.spec.ts [hunk 1]
```diff
@@ -387,4 +387,29 @@ describe('ValidationManager', () => {
...
+  describe('isPolygonValid', () => {
+    it('returns true when Polygon is valid', () => {
+      const footprint = createFootprint('Region');
+      const response = validationManager.isPolygonValid(footprint);
+      expect(response.isValid).toBe(true);
+    });
+
+    it('returns true when Polygon has BBOX', () => {
+      const footprint = createFootprint('Region');
+      footprint.bbox = [34.4077734887818, 31.454120054543704, 34.4780703169442, 31.5127352949048];
```
Comment (netanelC) [inline tests/unit/validator/validationManager.spec.ts:400]: This is hard-coded value, you can use faked generated value

File: tests/unit/validator/validationManager.spec.ts [hunk 2]
```diff
@@ -387,4 +387,29 @@ describe('ValidationManager', () => {
...
+      expect(response.isValid).toBe(true);
+    });
+
+    it('returns false when Polygon is invalid', () => {
+      const footprint = createFootprint('Region');
+      footprint.coordinates = [][0] as unknown as Position[][];
+      const response = validationManager.isPolygonValid(footprint);
+      expect(response.isValid).toBe(false);
+      expect(response.message).toContain(
+        `Invalid polygon provided. Must be in a GeoJson format of a Polygon. Should contain "type", "coordinates" and "BBOX" only.`
```
Comment (netanelC) [inline tests/unit/validator/validationManager.spec.ts:411]: Separate these messages or change the function from "toContain" to "toBe"

**Review by netanelC — CHANGES_REQUESTED** [review]
As you remove additional properties, did you check that the schema rejects json with unknown property (like: "avi")?

**Comment by netanelC** [comment]
I'm not sure this is a "fix" PR, this is more of a "feat"

**Review by geneg — APPROVED** [review]
Good Job!!!

File: tests/unit/validator/validationManager.spec.ts (hunk 1 above)
Comment (asafMasa) [reply tests/unit/validator/validationManager.spec.ts:400]: i want to use hard coded value as this is a test and it's need to be fixed

Comment (netanelC) [reply tests/unit/validator/validationManager.spec.ts:400]: Why? you can generate a valid bbox

File: tests/unit/validator/validationManager.spec.ts (hunk 2 above)
Comment (netanelC) [reply tests/unit/validator/validationManager.spec.ts:411]: Resolved F2F

**Pull request merged**